packages/core/src/components/workflow-steps/workflow-steps.tsx

```diff
@@ -103,21 +147,13 @@ export class WorkflowSteps {
         element.setAttribute('selected', 'true');
         this.stepSelected.emit(index);
       });
-      // const isEnabled = element.getAttribute('first');
-      // if(isEnabled){
-
-      // }
-      // console.log(isEnabled)
-      // const isDisabled = element.getAttribute('disabled') !== null;
-      // if (!isDisabled) element.addEventListener('click', () => '');
-      //element.addEventListener('mousedown', event => this.clicked(element, index));
     });
   }
 
   render() {
     return (
       <Host>
-        <div class={{ steps: true, vertical: this.vertical }}>
+        <div id="stepList" class={{ steps: true, vertical: this.vertical }}>
```

Comment:
We can not use `id` with an static string as an identifier because if someone uses multiple workflow-steps in one page its exist two `id`s which are the same https://www.w3.org/TR/html401/struct/global.html#adef-id.


You can use here

```
this. hostElement.queryElement(".steps")
```